pulp_2to3_migration/app/plugin/rpm/migrator.py

```diff
@@ -151,8 +155,9 @@ def pipeline_stages(self):
             QueryExistingContents(),
             ContentSaver(),
             RemoteArtifactSaver(),
-            InterrelateContent(),
             RelatePulp2to3Content(),
+            InterrelateContent(),
```

Comment:
Is it possible to avoid having any stage after `RelatePulp2to3Content`?
The presence of pulp3 content is a confirmation that content unit is fully migrated.
Having anything after that potentially brings correctness problems.

Reply:
I understand that. I have not figure any better way yet, see here https://github.com/pulp/pulp-2to3-migration/pull/132/commits/e97a63d1259716925a17510ef2d2319174f69dcb#diff-347b3cc7081a32fd6130c25dd18f31f3R355
how am i supposed to find pulp3 content based on pulp2_unit_id if they are not related?
I could find rpm package in mongo by unit_id, and then use it's NEVRA to find it in pulp3. But then, we have again issue with different checksums :)
Even if we switch the stages, this statement is still correct:
`The presence of pulp3 content is a confirmation that content unit is fully migrated.`
The unit itself, it migrated, it just that relations are not created yet.

Reply:
Yeah, though content is not complete without its relations. At least I think about it this way.

Does the fact that we migrate content in a specific order help here?
By the time pkg groups are migrated, packages are definitely fully migrated by that time.
What am I missing?

Reply:
@goosemania in that case i need to change the dc content from from self.batches to self.items, so we release the dc to the next stage